ramp4-pcar4/story-ramp
# Add stylesheet support

### Related Item(s)
#467 

### Changes
- [FEATURE] Adds support for adding css files through the config
- [FEATURE] Adds property `cssClasses` to panel configs for classes to add to the panel container
- [FIX] Add missing `customStyles` property to panel schema

### Testing
Steps:
1. Open default test page
2. Scroll to dynamic test panel, you should see an outline on the right side text panel. This is added by the stylesheet specified in the config. The style also targets a class added by the `cssClasses` property.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ramp4-pcar4/story-ramp/489)
